buildenv

```diff
@@ -0,0 +1,40 @@
+export ZOPEN_STABLE_URL="https://github.com/confluentinc/libserdes.git"
+export ZOPEN_STABLE_DEPS="zoslib pkgconfig make grep avro jansson coreutils which curl librdkafka libbsd openssl zlib zstd lz4"
+export ZOPEN_BUILD_LINE="STABLE"
+export ZOPEN_COMP="clang"
+
+export ZOPEN_EXTRA_CFLAGS="-DNSIG=42 -D_ALL_SOURCE -D_OPEN_SYS_FILE_EXT=1 -D_AE_BIMODAL=1 -D_ENHANCED_ASCII_EXT=0xFFFFFFFF -D_OPEN_SYS_SOCK_IPV6 -D_UNIX03_SOURCE -D_UNIX03_THREADS -D_XOPEN_SOURCE_EXTENDED=1 -D_POSIX_C_SOURCE=200112L -D_POSIX_SOURCE -DPOSIX_SIGNALS -D_XOPEN_SOURCE=600 -fzos-le-char-mode=ascii -std=gnu11 -mnocsect -fno-short-enums -Wno-error -mzos-target=zosv2r4" 
```

Comment:
you can probably remove -mzos-target=zosv2r4, -fzos-le-char-mode=ascii and others as they are defined in https://github.com/ZOSOpenTools/comp_clangport/blob/main/buildenv#L84